### PR TITLE
add countdown timer and update landing page

### PR DESCRIPTION
**Pull request by abhishall**

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -1,7 +1,21 @@
...
-  },
-  variants: {},
-  plugins: []
-}
+    theme: {
+        extend: {},
+        fontSize: {
+            xs: ".75rem",
+            sm: ".875rem",
+            tiny: ".875rem",
```
[the remark this answers is not in the document]
Comment (abhishall) [reply tailwind.config.js]: I just wanted to add the 7xl size. If I add only the 7xl size to the fontSize object in tailwind.config.js, all other sizes are gone.

[the remark this answers is not in the document]
Comment (rumansaleem) [reply tailwind.config.js]: you should specify `fontSize` in the `extend` configuration, that way, default config and your config will be merged. You wont lose any defaults. This applies to all theme configuration options.

[the remark this answers is not in the document]
Comment (abhishall) [reply tailwind.config.js]: alright.

File: tailwind.config.js [hunk 2]
```diff
@@ -1,7 +1,9 @@
...
-  plugins: []
-}
+    theme: {
+        extend: {},
+    },
+    variants: {},
+    plugins: []
+};
+
+// fontSize["7xl"] = "rem";
```
Comment (rumansaleem) [inline tailwind.config.js]: you can add extra font styles in:

```js
theme: {
    extend: {
         fontSize: {
             '7xl': '8rem' // add any extra styles here.
         }
    }
}
```

**Comment by rumansaleem** [comment]
@abhishall please resolve the merge conflicts. We do need that "enter game" link.

**Pull request closed without merging**